README.md

```diff
@@ -53,52 +76,56 @@ The `apple_search_ads_using_search_terms` variable below refers to the `search_t
 config-version: 2
 
 vars:
-  apple_search_ads__using_search_terms: false # Disable if you do not have the search_term_report table, or if you do not want search_term_report related metrics reported
-
+  apple_search_ads__using_search_terms: true # enable if you want to include the search_term_report table and want search_term_report related metrics reported
 ```
 
-### Change the Build Schema
-By default, this package builds the Apple Search Ads staging models within a schema titled (<target_schema> + `_stg_apple_search_ads`) in your target database. If this is not where you would like your Apple Search Ads staging data to be written to, add the following configuration to your `dbt_project.yml` file:
+### Change the build schema
+By default, this package builds the Apple Search Ads staging models within a schema titled (`<target_schema>` + `_apple_search_ads_source`) in your destination. If this is not where you would like your apple_search_ads staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
 models:
     apple_search_ads_source:
       +schema: my_new_schema_name # leave blank for just the target_schema
 ```
+    
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_apple_search_ads_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

Comment:
This link seems to be broken.

Reply:
Hmm, that's strange. It looks like it works for me? Did you click from vscode or did you copy and paste the URL? (both work for me) 🤔

Reply:
Strange... it works for me now. Something was probably a funk when I reviewed it on my side haha all good now!